11-query/04-delete-query.md

```diff
@@ -46,39 +48,81 @@ delete $emp;
 GraqlDelete query = Graql.match(
   var("org").isa("organisation").has("name", "Pharos"),
   var("emp").isa("employment").rel("employer", "org").rel("employee", "p")
-).delete("emp");
+).delete(var("emp").isa("employment"));
 ```
 [tab:end]
 </div>
 
 This deletes all instances of the `employment` type where the `employer` is an `organisation` with `name` of `"Pharos"`.
 
-## Delete Associations with Attributes
-Attributes with the same value are shared among their owners. It's important that one understands thoroughly how [attributes are defined](../09-schema/01-concepts.md#define-an-attribute) in a Grakn knowledge graph prior to performing `delete` queries on them.
+Attributes are normally owned by other concepts (ie. `$x has attribute $a`). This means we can both delete an attribute
+ownership, as in the next section, or delete the instance itself following the above style of using `isa` in the delete block.
```

Comment:
we have never used the term `block` as you have used in `the delete black` at the end of the sentence here.

Reply:
oke, changed to clause